libs/community/langchain_community/vectorstores/yellowbrick.py

```diff
@@ -24,19 +27,26 @@
 
 class Yellowbrick(VectorStore):
     """Yellowbrick as a vector database.
-
     Example:
         .. code-block:: python
             from langchain_community.vectorstores import Yellowbrick
             from langchain_community.embeddings.openai import OpenAIEmbeddings
             ...
     """
 
+    LSH_INDEX_TABLE: str = "_lsh_index"
```

Comment:
Could we remove class level attributes? You can move them into global scope or tuck them into the __init__

Reply:
Yes, I've moved into init